### DISCUSSION TITLE
-gaussian-blur 0.05 alternative

### DISCUSSION THREAD

**Discussion by tomasr78** [dlemstra/Magick.NET#960] [Help]
The ImageMagick has `-gaussian-blur 0.05` (`-gaussian-blur radius{xsigma}`) and Magick.Net has the same but with two parameters 
`public void GaussianBlur(double radius, double sigma)`. 

Should I code like this?

`GaussianBlur(0.05,0)`

**Comment by dlemstra** [comment]
The default value for `sigma` is `1.0` when it is not specified. You can use that for now but I will add an extra overload for this in the next release.

**Reply by dlemstra** [reply]
The new release that includes this overload has just been published.
